playbooks/ha_log_cfg_sync.yaml

```diff
@@ -25,16 +25,21 @@
         dest: /etc/rsyncd/rsyncd.conf
 
     - name: Start rsync daemon on slave nodes
+      template:
+        src: "{{ labkeeper_config_git_dest }}/rsync/rsyncd.service"
```

Comment:
it is better to name the file to xxx.j2, even if this can also work, but looks werid.